src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Runtime/BasicPreferStreamAsyncMemoryOverloads.Fixer.vb

```diff
@@ -99,5 +133,46 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Runtime
 
         End Function
 
+        Protected Overrides Function GetNodeWithNullability(invocation As IInvocationOperation) As SyntaxNode
```

Comment:
[cosmetic][suggestion] I would remove the extra blank lines in the rest of this class.

Reply:
Done.